api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -467,7 +468,6 @@ private void verifyInvitationKey(String invitationKey){
     if (updatedProfile.getContactEmail() != null) {
       if (!updatedProfile.getContactEmail().equals(user.getContactEmail())) {
         mailChimpService.addUserContactEmail(updatedProfile.getContactEmail());
```

Comment:
Is this still needed? Are we still putting researchers on the mailchimp list?

Reply:
No, and I plan on removing it in a separate PR.

Reply:
OK - sounds good